draft-normington-jsonpath-latest.md

```diff
@@ -84,43 +108,266 @@ informative:
 
 normative:
   RFC3629:
-  RFC5234:
+  RFC5234: abnf
+  RFC8259: json
 
 --- abstract
 
-
 JSONPath defines a string syntax for identifying values
 within a JavaScript Object Notation (JSON) document.
 
 --- note_
 
 **This document is a work in progress and has not yet been published
-as an Internet Draft.**
+as an Internet Draft** (which needs to be fixed soon).
+
+--- note_Contributing
+
+This document picks up the popular JSONPath specification dated
+2007-02-21 and provides a more normative definition for it.
+In its current state, it is a strawman document showing what needs to
+be covered.
+
+Comments and issues can be directed at the github repository *insert
+repo here* as well as (for the time when the more permanent home is
+being decided) at the dispatch@ietf.org mailing list.
 
 --- middle
 
 # Introduction
 
-JSONPath was introduced by Stefan Goessner as a simple
-form of XPath for JSON.
-See his original article {{Goessner}}.
+This document picks up the popular JSONPath specification dated
+2007-02-21 {{JSONPath-orig}} and provides a more normative definition for it.
+In its current state, it is a strawman document showing what needs to
+be covered.
 
 JSON is defined by {{RFC8259}}.
 
-## Requirements Language
+JSONPath is not intended as a replacement, but as a more powerful
+companion, to JSON Pointer {{RFC6901}}. \[insert reference to section
+where the relationship is detailed.]
```

Comment:
Referencing Pointer is good, but I wonder if we can do more. The purposes of the two syntaxes are different.  Pointer is for isolating a single location within a document. Path is a query syntax which pulls multiple locations.

Reply:
I'm not trying to address this in this version yet, so I just added a note.